src/Doctrine/Repository/ProjectRepository.php

```diff
@@ -78,11 +79,29 @@ public function load(ProjectId $projectId): Project
         }
     }
 
+    /**
+     * @param ProjectId $projectId
+     * @return bool
+     */
+    public function remove(ProjectId $projectId): bool
+    {
+        //TODO: some validation before remove maybe?
```

Comment:
If there is no project with given id query won't do anything. Validation of project Id is made in ProjectId ValueObject so we are here good without validation.